presentation/src/main/java/com/architect/coders/mu8/ui/activity/CategoriesActivity.kt

```diff
@@ -42,14 +47,17 @@ class MainActivity : AppCompatActivity() {
         viewModel.model.observe(this, Observer(::updateUi))
     }
 
-    private fun updateUi(model: UiModel) {
-        when (model) {
-            is UiModel.Content -> {
-                adapter = CategoriesAdapter(viewModel::onMoviewClicked, model.categories)
-                recycler.layoutManager = LinearLayoutManager(this)
-                recycler.adapter = adapter
-            }
-            is UiModel.Navigation -> Toast.makeText(this, model.category.title, Toast.LENGTH_SHORT).show()
+    private fun updateUi(model: UiModel) = when (model) {
+        is UiModel.Content -> {
+            adapter = CategoriesAdapter(viewModel::onCategoryClicked, model.categories)
+            recycler.layoutManager = LinearLayoutManager(this)
+            recycler.adapter = adapter
+        }
+        is UiModel.Navigation -> {
+            if (model.category.title == getString(R.string.category_section_comics))
```

Comment:
We have a different implementation here, please check my PR and let's sync up about this.

Reply:
@recaldev Cuple Questions ... 
1 )When I must to use a String Resource or just in a constant to save a constant value on the presentation module?

Reply:
done